src/views/SidebarView.vue

```diff
@@ -200,7 +200,9 @@ export default Vue.extend({
       return this.$store.state.genderOptions
     },
     assessments () {
-      return Object.values(this.$store.state.assessments).map((i) => ({ value: i.id, text: i.name }))
+      return Object.values(this.$store.state.assessments)
```

Comment:
is there a reason why we access the state directly in this app instead of using mapState from vuex?